### PR TITLE
Fixed: Resolved some PHP NOTICE about undefined variables #2650

### PR DESCRIPTION
**Pull request by ddb4github**
Like #2650 described

### CODE REVIEW

File: lib/aggregate.php [hunk 1]
```diff
@@ -506,15 +506,15 @@ function aggregate_change_graph_type($graph_index, $old_graph_type, $new_graph_t
 			 * any AREA/STACKed graph needs a base AREA entry
 			 * but e.g. a graph that prints on both negative and positive y-axis may hold two AREAs
 			 * so it's a good idea to keep all AREA entries of the first aggregated elementary graph (index 0)*/
-			if ($graph_index == 0 && $item_index == 1 &&
+			if (($graph_index == 0 || $graph_index == 1) &&
```
Comment (netniV) [inline lib/aggregate.php]: This isn’t just a missing variable change, it’s changed the logic.

Comment (ddb4github) [reply lib/aggregate.php]: From comments above this line.  logic like to allow both of 0 and 1 for graph_index

Comment (ddb4github) [reply lib/aggregate.php]: aha!  incorrect fixing for global_language.php, should add 'global $l10n;

File: aggregate_graphs.php [hunk 2]
```diff
@@ -951,7 +951,7 @@ function changeTotalsType() {
 						$form_array[$field_name]['value']   = (isset($graphs) ? $graphs[$field_name] : '');
 						$form_array[$field_name]['form_id'] = (isset($graphs) ? $graphs['id'] : '0');
 
-						if (!(($use_graph_template == false) || ($graphs_template['t_' . $field_name] == 'on'))) {
+						if (!(($use_graph_template == false) || ($form_array['t_' . $field_name] == 'on'))) {
```
Comment (cigamit) [inline aggregate_graphs.php]: Should be $graphs

**Comment by cigamit** [comment]
I have reviewed all the changes.  Great work @ddb4github!

**Pull request merged**